doc/changes.rst

```diff
@@ -1,6 +1,81 @@
 Changelog
 =========
 
+Release v0.2.1 (2020/11/14)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4253459.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.4253459
+
+Highlights
+
+* 🎉 **Patch release with more tutorials and gallery examples!** 🎉
+* Support Python 3.9 (`#689 <https://github.com/GenericMappingTools/pygmt/pull/689>`__)
```

Comment:
```suggestion
* 🐍 Support Python 3.9 (`#689 <https://github.com/GenericMappingTools/pygmt/pull/689>`__)
```
Use a snake emoji here?

Reply:
Great! Tried to search for "Python" but failed. :snake: looks good.